ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/advanced-gas-fee-inputs.util.js

```diff
@@ -0,0 +1,24 @@
+import { uniq } from 'lodash';
+import { toBigNumber } from '../../../../../shared/modules/conversion.utils';
+
+export function roundToDecimalPlacesRemovingExtraZeroes(
+  numberish,
+  numberOfDecimalPlaces,
+) {
+  if (numberish) {
+    return toBigNumber.dec(
+      toBigNumber.dec(numberish).toFixed(numberOfDecimalPlaces),
+    );
+  }
+  return null;
+}
+
+export const renderFeeRange = (feeRange) => {
+  if (feeRange) {
+    const formattedRange = uniq(
+      feeRange.map((fee) => roundToDecimalPlacesRemovingExtraZeroes(fee, 2)),
+    ).join(' - ');
+    return `${formattedRange} GWEI`;
+  }
+  return null;
+};
```

Comment:
File can be named `utils.js` I think

Reply:
We have both `.util.js` and `.utils.js` in our repo, for now, the current one seems to be okay. 🤞